src/Workspaces/Core/Portable/Utilities/SpellChecker.cs

```diff
@@ -11,7 +11,7 @@ namespace Roslyn.Utilities
 {
     internal class SpellChecker
     {
-        private const string SerializationFormat = "1";
+        private const string SerializationFormat = "2";
```

Comment:
Is this just to invalidate old trees with 10k single-character items?  If so, won't the recursion check above prevent the SO?